### ISSUE TITLE
执行sh scripts/run_softmax_ner.sh退出

### ISSUE THREAD

**Issue by Xuanfang1121** [liuyukid/transformers-ner#18]
执行sh退出

> 7f6daa2f6000-7f6daa2f7000 r--p 00008000 08:11 39198496                   /opt/nlu/conda_env_dir/torchgpu/lib/python3.6/lib-dynload/math.cpython-36m-x86_64-linux-gnu.so
7f6daa2f7000-7f6daa2f9000 rw-p 00009000 08:11 39198496                   /opt/nlu/conda_env_dir/torchgpu/lib/python3.6/lib-dynload/math.cpython-36m-x86_64-linux-gnu.so
7f6daa2f9000-7f6daa2fc000 r--p 00000000 08:11 39198484                   /opt/nlu/conda_env_dir/torchgpu/lib/python3.6/lib-dynload/_struct.cpython-36m-x86_64-linux-gnu.so
7f6daa2fc000-7f6daa301000 r-xp 00003000 08:11 39198484                   /opt/nlu/conda_env_dir/torchgpu/lib/python3.6/lib-dynload/_struct.cpython-36m-x86_64-linux-gnu.so
7f6daa301000-7f6daa303000 r--p 00008000 08:11 39198484                   /opt/nlu/conda_env_dir/torchgpu/lib/python3.6/lib-dynload/_struct.cpython-36m-x86_64-linux-gnu.so
7f6daa303000-7f6daa304000 ---p 0000a000 08:11 39198484                   /opt/nlu/conda_env_dir/torchgpu/lib/python3.6/lib-dynload/_struct.cpython-36m-x86_64-linux-gnu.so
7f6daa304000-7f6daa305000 r--p 0000a000 08:11 39198484                   /opt/nlu/conda_env_dir/torchgpu/lib/python3.6/lib-dynload/_struct.cpython-36m-x86_64-linux-gnu.so
7f6daa305000-7f6daa307000 rw-p 0000b000 08:11 39198484                   /opt/nlu/conda_env_dir/torchgpu/lib/python3.6/lib-dynload/_struct.cpython-36m-x86_64-linux-gnu.so
7f6daa307000-7f6daa48c000 rw-p 00000000 00:00 0 
7f6daa48e000-7f6daa48f000 rw-p 00000000 00:00 0 
7f6daa48f000-7f6daa490000 rwxp 00000000 00:00 0 
7f6daa490000-7f6daa497000 r--s 00000000 fd:00 67724065                   /usr/lib64/gconv/gconv-modules.cache
7f6daa497000-7f6daa498000 rw-p 00000000 00:00 0 
7f6daa498000-7f6daa499000 r--p 00020000 fd:00 2307900                    /usr/lib64/ld-2.18.so
7f6daa499000-7f6daa49a000 rw-p 00021000 fd:00 2307900                    /usr/lib64/ld-2.18.so
7f6daa49a000-7f6daa49b000 rw-p 00000000 00:00 0 
7ffd0c5ac000-7ffd0c5cd000 rw-p 00000000 00:00 0                          [stack]
7ffd0c5fe000-7ffd0c600000 r-xp 00000000 00:00 0                          [vdso]
ffffffffff600000-ffffffffff601000 r-xp 00000000 00:00 0                  [vsyscall]
run_softmax_ner.sh: line 28: 11118 Aborted                 (core dumped) CUDA_VISIBLE_DEVICES='0' python3 ./examples/run_softmax_ner.py --data_dir $DATA_DIR --model_type $MODEL_MODEL_TYPE --model_name_or_path $MODEL_NAME_OR_PATH --output_dir $OUTPUT_DIR --labels $LABEL --overwrite_output_dir --do_train --do_eval --evaluate_during_training --adv_training fgm --num_train_epochs 3 --max_seq_length 128 --logging_steps 0.2 --per_gpu_train_batch_size 8 --per_gpu_eval_batch_size 8 --learning_rate 5e-5 --bert_lr 5e-5 --classifier_lr 5e-5

这种错误如何解决？
